apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -78,7 +79,7 @@ class VotePanelContent extends React.Component {
     if (tokenContract && user) {
       tokenContract
         .balanceOfAt(user, vote.data.snapshotBlock)
-        .first()
+        .pipe(first())
```

Comment:
This is an external call with a single response. `pipe` and `first` is redundant. If you like, you can also convert to a promise (just syntactic sugar here)